ci-operator/step-registry/azure/provision/disk-encryption-set/azure-provision-disk-encryption-set-commands.sh

```diff
@@ -75,12 +75,27 @@ keyvault="${NAMESPACE}-${UNIQUE_HASH}-kv"
 keyvault_key="${NAMESPACE}-${UNIQUE_HASH}-kvkey"
 des="${NAMESPACE}-${UNIQUE_HASH}-des"
 create_disk_encryption_set ${RESOURCE_GROUP} ${keyvault} ${keyvault_key} ${des}
+des_list="${des}"
 
-echo "Granting service principal reader permissions to the DiskEncryptionSet: ${des}"
-des_id=$(az disk-encryption-set show -n ${des} -g ${RESOURCE_GROUP} --query "[id]" -o tsv)
-cluster_sp_id=$(cat "${AZURE_AUTH_LOCATION}" | jq -r ".clientId")
-run_command "az role assignment create --assignee ${cluster_sp_id} --role Owner --scope ${des_id} -o jsonc"
+if [[ "${SET_DIFFERENT_DES_FOR_COMPUTE}" == "true" ]]; then
```

Comment:
Some days ago ibmcloud did some similar functions, refer to https://github.com/openshift/release/pull/47656, that means, once any of `ENABLE_DES_DEFAULT_MACHINE`, `ENABLE_DES_COMPUTE`, and `ENABLE_DES_CONTROL_PLANE` is enabled, always create a dedicated DES, than we do not need to introduce `SET_DIFFERENT_DES_FOR_COMPUTE` any more.

Reply:
updated.
seems that OCP-51422 is duplicated, using same des in controlePlane and compute (it has been covered when only setting `ENABLE_DES_DEFAULT_MACHINE`), I will disactive it and remove from 4.15 full function test run.